### ISSUE TITLE
mark repo as abandoned?

### ISSUE THREAD

**Issue by Chris53897** [kbond/ZenstruckFormBundle#44]
Maybe it is time to mark that repo as abandoned and archived?

**Comment by kbond** [comment]
I agree but I still do maintain a [compat branch](https://github.com/kbond/ZenstruckFormBundle/tree/compat-sf5) that I keep updated as it's still used in a legacy app of mine. It's on my list to remove from this app - I'll abandon once this is done.
